driver/src/main/scala/com/stratio/crossdata/driver/HttpDriver.scala

```diff
@@ -80,22 +97,37 @@ class HttpDriver private[driver](driverConf: DriverConf,
 
   }
 
-  protected[driver] def openSession(): Try[Boolean] = ???
-  /*{
+  protected[driver] def openSession(): Try[Boolean] = {
 
-    val result = simpleRequest(
+    val response = simpleRequest(
       securitizeCommand(OpenSessionCommand()),
-      "openSession",
-      (opSessionReply: OpenSessionReply) => opSessionReply.isOpen
+      "query",
```

Comment:
Indeed, it must since que command is designed as a query.  It can be be placed under `sessions` endpoint by duplicating some code and changing the original services schema.